app/models/ResourceLinks.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models
+
+import play.api.libs.json.JsObject
+import play.api.libs.json.Json
+import play.api.libs.json.OFormat
+
+case class ResourceLinks(_links: Option[Map[String, JsObject]], metaData: Option[MetaData])
```

Comment:
Why is `_links` and metaData optional?

Reply:
Not sure, ill investigate and revert if needed

Reply:
I wasn't sure if brining back no links would be valid so made this as an Option. Also, I wasn't 100% sure we'd be using MetaData and allowing limiting results by version or snapshot date